frontend/src/modules/Environments/components/EnvironmentFeatures.js

```diff
@@ -7,119 +9,73 @@ import {
   ListItem,
   Typography
 } from '@mui/material';
-import PropTypes from 'prop-types';
-import React from 'react';
 import { Label } from 'design';
+import { ModuleNames, isModuleEnabled } from 'utils';
 
 export const EnvironmentFeatures = (props) => {
   const { environment, ...other } = props;
 
+  // Define your list of features/modules with additional attributes
+  const features = [
+    {
+      title: 'Dashboards',
+      env: 'dashboardsEnabled',
```

Comment:
Very minor: can we rename `env` as something that indicates that it is the env_parameter?